ios/ImageCropPicker.h

```diff
@@ -22,7 +22,7 @@
 #import "QBImagePicker.h"
 #import "RSKImageCropper.h"
 #else
```

Comment:
To handle cases where this library is not installed using cocoapods we should include another conditon

```
#elseif __has_include("QBImagePicker/QBImagePicker.h")
#import "QBImagePicker/QBImagePicker.h"
#import <RSKImageCropper/RSKImageCropper.h>

```